R/get_athena_query_response.R

```diff
@@ -10,6 +10,8 @@
 #'
 #'@param sql_query A string specifying the SQL query you want to send to athena. See packages github readme for info on the flavour of SQL Athena uses.
 #'
+#'@param out_path The s3 folder the query results will be sent to e.g. s3://my-bucket/__temp__
```

Comment:
Add line that says must have read/write accessibility to path provided?

Reply:
Yep - makes sense - done